install_config/aggregate_logging.adoc

```diff
@@ -134,7 +134,7 @@ endif::openshift-enterprise[]
 |`openshift_logging_use_ops`
 |If set to `true`, configures a second Elasticsearch cluster and Kibana for
 operations logs. Fluentd splits logs between the main cluster and a cluster reserved
-for operations logs (which consists of *_/var/log/messages_* on nodes and the logs from the
+for operations logs (which consists of the logs from the
 projects *default*, *openshift*, and *openshift-infra*). This means a second Elasticsearch
```

Comment:
`projects *default*, *openshift*, and *openshift-infra*`, as well as docker, openshift, and system logs from the journal).